tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/core/SentenceTransform.scala

```diff
@@ -65,12 +75,86 @@ case object FactorieSentenceTagger extends SentenceTransform {
       defaultPostagger.postagTokenized(nlpStackTokens).toIndexedSeq
     Sentence(NexusToken +: (taggedTokens.zip(sentence.tokens.tail) map {
       case (tagged, untagged) =>
+        val autoPos = Symbol(tagged.postag)
+        val autoCpos = Symbol(WordClusters.ptbToUniversalPosTag.getOrElse(
+          autoPos.name, "X"
+        ))
         untagged.updateProperties(Map(
-          'factoriePos -> Set(Symbol(tagged.postag)),
-          'factorieCpos -> Set(Symbol(WordClusters.ptbToUniversalPosTag.getOrElse(
-            tagged.postag, "X"
-          )))
+          'autoPos -> Set(autoPos),
+          'autoCpos -> Set(autoCpos)
+        ))
+    }))
+  }
+}
+
+/** The StanfordSentenceTagger tags an input sentence with automatic part-of-speech tags
+  * from the Stanford tagger.
+  */
+case object StanfordSentenceTagger extends SentenceTransform {
+
+  @transient private val stanfordTagger = new StanfordPostagger()
+
+  def transform(sentence: Sentence): Sentence = {
+    val words: IndexedSeq[String] = sentence.tokens.tail map { tok => tok.word.name }
+    val nlpStackTokens: IndexedSeq[NLPStackToken] =
+      Tokenizer.computeOffsets(words, words.mkString).toIndexedSeq
+    val taggedTokens: IndexedSeq[PostaggedToken] =
+      stanfordTagger.postagTokenized(nlpStackTokens).toIndexedSeq
+    Sentence(NexusToken +: (taggedTokens.zip(sentence.tokens.tail) map {
+      case (tagged, untagged) =>
+        val autoPos = Symbol(tagged.postag)
+        val autoCpos = Symbol(WordClusters.ptbToUniversalPosTag.getOrElse(
+          autoPos.name, "X"
+        ))
+        untagged.updateProperties(Map(
+          'autoPos -> Set(autoPos),
+          'autoCpos -> Set(autoCpos)
+        ))
+    }))
+  }
+}
+
+/** The MultiSentenceTagger tags an input sentence with automatic part-of-speech tags
+  * from the Stanford tagger.
+  */
+case object MultiSentenceTagger extends SentenceTransform {
```

Comment:
`MultiSentenceTagger` sounds like it tags multiple sentences. Rename? To `SentenceMultiTagger` or something?